src/content/pages/faq.mdx

```diff
@@ -12,6 +12,8 @@ subtitle: Frequently Asked Questions about EuroPython
   <li>**10 January 2025** [Call for Proposals (CfP)](/programme/cfp/) opens</li>
   <li>**11 January 2025** [Speaker Mentorship Programme](/programme/mentorship/) sign-up opens</li>
   <li>**21 January 2025:** Speaker Mentorship signup closes</li>
+  <li>**30 January 2025:** [Call for Reviewers](https://forms.gle/4GTJjwZ1nHBGetM18) & review process begin</li>
```

Comment:
```suggestion
  <li>**30 January 2025:** [Call for Reviewers](https://forms.gle/4GTJjwZ1nHBGetM18) & review process begins</li>
```
Can I get an "s"? Just to be in  line with the other line endings?

Reply:
No, because there are two things, so the subject is not singular :)